homeassistant/components/tibber/sensor.py

```diff
@@ -411,6 +415,14 @@ async def async_update(self) -> None:
         ):
             return
 
+        if now.hour == 0:
+            self._attr_extra_state_attributes["yesterday"] = (
```

Comment:
We're trying to avoid state attributes in general and we don't allow setting state that isn't the current state.